docs/source/_templates/sdk_submit_tutorial.rst

```diff
@@ -243,92 +240,103 @@ dependent on any others, they cannot be submitted directly to the validator.
 
-    const {BatchEncoder} = require('sawtooth-sdk')
+    const transactions = [transaction]
 
-    const batcher = new BatchEncoder(privateKey)
+    const batchHeaderBytes = protobuf.BatchHeader.encode({
+        signerPublicKey: signer.getPublicKey().asHex(),
+        transactionIds: transactions.map((txn) => txn.headerSignature),
+    }).finish()
 
 {% else %}
 
 .. code-block:: python
 
-    from sawtooth_sdk.client.encoding import BatchEncoder
+    from sawtooth_sdk.protobuf.batch_pb2 import BatchHeader
+
+    txns = [txn]
 
-    batcher = BatchEncoder(private_key)
+    batch_header_bytes = BatchHeader(
+        signer_public_key=signer.get_public_key().as_hex(),
+        transaction_ids=[txn.header_signature for txn in txns],
+    ).SerializeToString()
 
 {% endif %}
 
 
 2. Create the Batch
 -------------------
 
-Using the SDK, creating a Batch is as simple as calling the *create* method and
-passing it one or more Transactions. If serialized, there is no need to
-decode them first. In addition to Transaction instances, the BatchEncoder can
-handle TransactionLists encoded as both raw binaries and url-safe base64
-strings.
-
+Using the SDK, creating a Batch is similar to creating a transaction.  The
+header is signed, and the resulting signature acts as the Batch's ID.  The Batch
+is then constructed out of the header bytes, the header signature, and the
+transactions that make up the batch.
 
 {% if language == 'JavaScript' %}
 
 .. code-block:: javascript
 
-    const batch = batcher.create(txn)
-
-    const batch2 = batcher.create([txn, txn2])
-
-    const batch3 = batcher.create(txnBytes)
+    const signature = signer.sign(batchHeaderBytes)
 
+    const batch = protobuf.Batch.create({
+        header: batchHeaderBytes,
+        headerSignature: signature,
+        transactions: transactions
+    }
 
 {% else %}
 
 .. code-block:: python
 
-    batch = batcher.create(txn)
+    from sawtooth_sdk.protobuf.batch_pb2 import Batch
 
-    batch2 = batcher.create([txn, txn2])
+    signature = signer.sign(batch_header_bytes)
 
-    batch3 = batcher.create(txn_bytes)
+    batch = Batch(
+        header=batch_header_bytes,
+        header_signature=signature,
+        transactions=txns
+    )
 
 {% endif %}
 
 
 3. Encode the Batch(es) in a BatchList
 --------------------------------------
 
-Like the TransactionEncoder, BatchEncoders have both *encode* and
-*createEncoded* methods for serializing Batches in a BatchList. If encoding
-multiple Batches in one BatchList, they must be created individually first, and
-then encoded. If only wrapping one Batch per BatchList, creating and encoding
-can happen in one step.
-
+In order to submit Batches to the validator, they  must be collected into a
+*BatchList*.  Multiple batches can be submitted in one BatchList, though the
+Batches themselves don't necessarily need to depend on each other.  Likewise,
+there is no guarantee that additional batches (from other clients) may be
```

Comment:
Change the "Likewise" sentence (per our conversation):

   Unlike Batches, a BatchList is not atomic. Batches from other clients may be interleaved with yours.